conf/snowplow_monthlygood_checkout_event_schema_1_0_1.json

```diff
@@ -0,0 +1,39 @@
+{
+	"$schema": "https://raw.githubusercontent.com/kiva/snowplow/master/conf/snowplow_monthlygood_checkout_event_schema_1_0_1.json#",
```

Comment:
I believe the schema should be `http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#`
Having the spec define its schema as itself is recursive.

Reply:
ooof... all of our schemas do this 🤦

Reply:
Actually, I think these do point to themselves as they are "Self describing schemas"

ref. https://snowplowanalytics.com/blog/2014/05/15/introducing-self-describing-jsons/ (section 4.) + https://tools.ietf.org/html/draft-zyp-json-schema-04#section-5.6

Reply:
Well... I do so the part below that which references the url you've pasted above.... I'll add that .